src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -175,8 +175,14 @@ public static final class prefPitch {
     public static final SN_DoublePreference pitchP = new SN_DoublePreference("pitchP", 70);
     public static final SN_DoublePreference pitchI = new SN_DoublePreference("pitchI", 0);
     public static final SN_DoublePreference pitchD = new SN_DoublePreference("pitchD", 0);
+    public static final SN_DoublePreference pitchSupplyCurrentThreshold = new SN_DoublePreference(
+        "pitchSupplyCurrentThreshold", 0);
+    public static final SN_DoublePreference pitchWithSupplyTimeThreshold = new SN_DoublePreference(
+        "pitchWithSupplyTimeThreshold", 0);
     public static final SN_BooleanPreference pitchInvert = new SN_BooleanPreference("pitchInvert", true);
-
+    public static final SN_BooleanPreference enablePitchSupplyCurrentLimit = new SN_BooleanPreference(
```

Comment:
Can you keep the names consistent with the others? Maybe:
pitchSupplyCurrentLimitEnable

Reply:
nuh uh